doc/tutorials/3d/point_cloud/point_cloud.markdown

```diff
@@ -0,0 +1,75 @@
+Point cloud visualisation {#tutorial_point_cloud}
+==============================
+
+|    |    |
+| -: | :- |
+| Original author | Klepikov Dmitrii |
+| Compatibility | OpenCV >= 5.0 |
+
+Goal
+----
+
+In this tutorial you will:
+
+-   Load and save point cloud data
+-   Visualise your data
+
+Requirements
+------------
+
+For visualisations u need to complile OpenCV library with OpenGL support. For this you should set WITH_OPENGL flag on in cmake
```

Comment:
compile
>> in cmake
while building OpenCV from source.

Reply:
u -> you